README.md

```diff
@@ -117,3 +117,19 @@ By default, only _Deploy Previews_ (Pull Requests) are deployed to [Netlify][net
 [storybook]: https://storybook.js.org
 [netlify]: https://www.netlify.com
 [travis]: https://travis-ci.org/commercetools/ui-kit/
+
+## Translations (i18n)
+
+The UI Kit uses `react-intl`. The core messages are written down in `messages.js` files.
+
+The translations for the supported languages exist in the `/i18n` folder. We do not use any automated translation software.
```

Comment:
are there any plans to use it?

Reply:
https://github.com/commercetools/ui-kit/issues/46#issuecomment-421256695